### PR TITLE
Removing access to BSP endpoints in PROD

### PR DESCRIPTION
**Pull request by jakub-kozlowski**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BSP-140

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/prod/common/divorce/div-cos.yaml [hunk 1]
```diff
@@ -41,6 +41,8 @@ spec:
         DOCUMENTATION_SWAGGER_ENABLED: "false"
         FEATURE_RESP_SOLICITOR_DETAILS: "true"
         SCHEDULER_SEND_UPDATED_CASES_TO_ROBOTICS_CRON: "0 0 4 ? * * *"
+        BSP_SERVICES_ALLOWED_TO_UPDATE:
```
Comment (yarpo) [inline k8s/prod/common/divorce/div-cos.yaml]: Hm, maybe 

```
BSP_SERVICES_ALLOWED_TO_UPDATE: ""
```
?

Comment (jakub-kozlowski) [reply k8s/prod/common/divorce/div-cos.yaml]: that would be empty string, which is not the same as empty value - we won't know before we test :)

Comment (jakub-kozlowski) [reply k8s/prod/common/divorce/div-cos.yaml]: I'd try with empty value and only change it to empty string if it doesn't work

Comment (yarpo) [reply k8s/prod/common/divorce/div-cos.yaml]: yeah, but we use it this way:

`${BSP_SERVICES_ALLOWED_TO_UPDATE:default_value}`

if BSP_SERVICES_ALLOWED_TO_UPDATE is not set -> default_value. I guess.

**Pull request merged**